src/localization/online_localizer/online_localizer.cpp

```diff
@@ -52,12 +52,12 @@ OnlineLocalizer::OnlineLocalizer() {
   _currentBestHyp = source;
 }
 
-bool OnlineLocalizer::setSuccessorManager(SuccessorManager::Ptr succManager) {
-  if (!succManager) {
+bool OnlineLocalizer::setSuccessorManager(SuccessorManager *successorManager) {
+  if (!successorManager) {
     printf("[ERROR][OnlineLocalizer] Successor manager is not set\n");
```

Comment:
nit: You seem to be using glog in other places, so you might as well use it here.

Reply:
You are right, but this is a part of online_localizer which I would like to refactor in the next PRs altogether, very similar to what I am doing here with the `succesor_manager`.